src/com/mitrais/view/WelcomeScreen.java

```diff
@@ -0,0 +1,49 @@
+package src.com.mitrais.view;
+
+import src.com.mitrais.model.Account;
+import src.com.mitrais.utils.Helper;
+import src.com.mitrais.utils.Validation;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class WelcomeScreen extends BaseScreen {
+    private Scanner in = new Scanner(System.in);
+    private TransactionScreen transactionScreen = new TransactionScreen();
+    Validation validation = new Validation();
+
+    public void show(){
+        System.out.println("======================= ATM Simulation =======================");
+        System.out.println("Enter Account Number: ");
+
+        String accountNumber = in.nextLine();
+
+        List<String> errAccount = validation.ValidateAccountNumberOrPin(accountNumber, true);
```

Comment:
Please stick to the java convention for method and variable naming

Reply:
fixed